app/controllers/application_controller.rb

```diff
@@ -2,6 +2,10 @@ class ApplicationController < ActionController::Base
   include Clearance::Controller
   protect_from_forgery with: :exception
 
+  def disable_search
+    @disable_search = true
+  end
```

Comment:
I could create a method 
```
def enable_search
  @disable_search = false
end
```
but it just creates a before action, which I use to put on certain pages I don't want the search checkboxes on. 

Originally I just wanted it off the show pages, but I eventually added it to everything except the dashboard. I could just undo the partial for the search... but I'm torn because this leaves the option open if I fix the search boxes later on for non-dashboard pages.